src/lm/arpa-file-parser.h

```diff
@@ -86,9 +91,35 @@ class ArpaFileParser {
   void Read(std::istream &is, bool binary);
 
   /// Parser options.
-  const ArpaParseOptions& Options() const { return options_; }
+  const ArpaParseOptions &Options() const { return options_; }
```

Comment:
I should have been more clear-- for some reason the way most of the code is, that in function return types it's to the left, but to the right in declarations, e.g.
const ArpaParseOptions& Options(const ArpaParseOptions &foo);
Not that it really matters, but that's the way the code generally looks.